app/services/hyrax/workflow/notification_service.rb

```diff
@@ -51,10 +51,10 @@ def recipients(notification)
         case entity.workflow_state.name
         when 'pending_review'
           # notify the managers to review the deposit (CC depositors)
-          {
-            to: entity.workflow.permission_template.agent_ids_for(access: 'manage',  agent_type: 'user'),
-            cc: entity.workflow.permission_template.agent_ids_for(access: 'deposit', agent_type: 'user')
-          }
+          permissions = entity.workflow.permission_template
+          managers = Array.wrap(permissions.agent_ids_for(access: 'manage', agent_type: 'user'))
+          depositors = Array.wrap(permissions.agent_ids_for(access: 'deposit', agent_type: 'user'))
+          { 'to' => managers, 'cc' => depositors }
```

Comment:
Also wanted to checked that the `else` block returns a hash, so I tried something similar in the REPL and it looks like it does, i.e.
```ruby
> [*1..10].each_with_object({}) {|x, h| h['a'] ||= []; h['a'].push(x); }
=> {"a"=>[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]}
```